tests/end2end/ia3/clean.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
We should be able to remove this file as it is the same as the base clean file in the parent directory.